CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Changes by Version
 
+## v0.32.0 (2019-03-20)
+
+* Trace Identifiers added to `SpanContext`.
+* Finishing `Span` upon Scope close is deprecated overall.
+* `ScopeManager.active()` is deprecated.
+* `SpanBuilder.startActive()` is deprecated.
+* `AutoFinishScopeManager` is deprecated.
+* Added a new `Binary` format on top of `ByteBuffer`.
+* Added generic typed `setTag()`/`withTag()` versions.
+* Split Inject and Extract builtin interfaces.
+* `Tracer` implements `Closable`.
+* Added `GlobalTracer.registerIfAbsent()`.
+* Added `GlobalTracer.isRegistered()`.
```

Comment:
`GlobalTracer.isRegistered()` already existed. The implementation changed.

Reply:
Oh, right. Thanks for the review. Will update the Changelog ;)